lib/pages/forms_pages/adult_report_page.dart

```diff
@@ -299,6 +189,8 @@ class _AdultReportPageState extends State<AdultReportPage> {
     if (widget.editReport != null) {
       widget.loadData!();
     }
+    Navigator.pop(context);
+    Navigator.pop(context);
```

Comment:
Duplicated?

Reply:
In this case we need 2 Navigator.pop() to go back to the homepage. Since the camera is open, the first pop() goes to the CameraPage but with the camera closed, and the second one goes to homepage. Since CameraPage without the camera open is not useful, I just skip it

Reply:
understood, thank's for the clarification :smile: